include/read/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+file(MAKE_DIRECTORY "${CMAKE_SOURCE_DIR}/3rdparty/opencv/build")
+
+execute_process(
+    COMMAND ${CMAKE_COMMAND} -S "${CMAKE_SOURCE_DIR}/3rdparty/opencv" -B "${CMAKE_SOURCE_DIR}/3rdparty/opencv/build"
```

Comment:
```suggestion
    COMMAND ${CMAKE_COMMAND} -S "${CMAKE_SOURCE_DIR}/3rdparty/opencv" -B "${CMAKE_SOURCE_DIR}/3rdparty/opencv/build" -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
```

Reply:
-- 'Release' build type is used by default.

Reply:
Ok, it means when we build Debug build for our library, we will se Release OpenCV

Reply:
When we build Debug build for our library, does opencv also need to be build in Debug? But opencv is building first, how will it work?

Reply:
```
# Configure a release build
cmake -S . -B build/ -D CMAKE_BUILD_TYPE=Release
```
should work fine

Reply:
initial problem still not solved